### PR TITLE
Minor punctuation changes.

### PR DESCRIPTION
**Pull request by jzeferino**

### CODE REVIEW

**Comment by jzeferino** [comment]
I signed it!

File: interop.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ For example, Mockito's `when` function requires backticks when used from Kotlin:
 
 ```kotlin
 val callable = Mockito.mock(Callable::class.java)
-Mockito.`when`(callable.call()).thenReturn(/* … */)
+Mockito.when(callable.call()).thenReturn(/* … */)
```
Comment (dector) [inline interop.md]: `when` is a hard keyword in Kotlin. Backticks are required here. :)

Comment (jzeferino) [reply interop.md]: thank you. will revert it.

Comment (jzeferino) [reply interop.md]: @JakeWharton I'm no longer able to edit this PR, can you please just rever this change? The remaining changes are OK.

File: style.md [hunk 2]
```diff
@@ -255,8 +255,8 @@ The prime directive of line-wrapping is: prefer to break at a **higher syntactic
 
  1. When a line is broken at a _non-assignment_ operator the break comes _before_ the symbol.
     * This also applies to the following "operator-like" symbols:
-      * the dot separator (`.`)
-      * the two colons of a member reference (`::`)
+      * the dot separator (`.`).
+      * the two colons of a member reference (`::`).
```
Comment (JakeWharton) [inline style.md]: These two are not sentences and don't need a period.

**Comment by JakeWharton** [comment]
I have fixed two of the ones from this PR, but the others are not needed. Thanks.

**Pull request closed without merging**